### ISSUE TITLE
Remove easydict dependency

### ISSUE THREAD

**Issue by iizukak** [blue-oil/blueoil#1092]
We use easydict to access dictionary.
https://github.com/blue-oil/blueoil/blob/master/setup.cfg#L23

But the library longer maintenance(~2years).
https://pypi.org/project/easydict/

I want to remove library dependence.

**Comment by patrick-su** [comment]
@iizukak seems like we can close this?

**Comment by iizukak** [comment]
@patrick-su 
NO. @odoku san implement SmartDict class, but not remove easydict dependency.
